README.md

```diff
@@ -388,6 +388,92 @@ Once the MQ source connector has delivered messages to Kafka with exactly-once s
 The MQ source connector is designed to fail on start-up in certain cases to ensure that exactly-once delivery is not compromised.
 In some of these failure scenarios, it will be necessary for an MQ administrator to remove messages from the exactly-once state queue before the MQ source connector can start up and begin to deliver messages from the source queue again. In these cases, the MQ source connector will have the `FAILED` status and the Kafka Connect logs will describe any required administrative action.
 
+## Using the XML Record Builder
+
+The MQ Source Connector now supports processing JMS messages using an XML record builder. This builder parses XML payloads, validates them against an XSD schema, and produces Kafka records with structured data.
+
+### Prerequisites
+
+Download and install the XML converter JAR:
+
+```bash
+curl -L -o kafka-connect-xml-converter-{VERSION}.jar https://github.com/ibm-messaging/kafka-connect-xml-converter/releases/download/v{VERSION}/kafka-connect-xml-converter-{VERSION}-jar-with-dependencies.jar
+
+mvn install:install-file \
+  -Dfile=kafka-connect-xml-converter-{VERSION}-jar-with-dependencies.jar \
+  -DgroupId=com.ibm.eventstreams.kafkaconnect.plugins \
+  -DartifactId=kafka-connect-xml-converter \
+  -Dversion={VERSION} \
+  -Dpackaging=jar
+```
+
+Or, if using Maven:
+
+```xml
+<dependency>
+  <groupId>com.ibm.eventstreams.kafkaconnect.plugins</groupId>
+  <artifactId>kafka-connect-xml-converter</artifactId>
+  <version>{VERSION}</version>
+</dependency>
+```
+
+Replace `{VERSION}` with the desired version number.
+
+---
+
+### Connector Configuration
+
+To enable the XML record builder, add the following properties to your connector configuration:
+
+```java
+connectorConfigProps.put("mq.record.builder", "com.ibm.eventstreams.kafkaconnect.plugins.xml.XmlMQRecordBuilder");
```

Comment:
what is "connectorConfigProps"? 

Why wouldn't you configure it through normal connector config?